torch/distributed/elastic/agent/server/local_elastic_agent.py

```diff
@@ -254,6 +265,17 @@ def _start_workers(self, worker_group: WorkerGroup) -> Dict[int, Any]:
             if "OMP_NUM_THREADS" in os.environ:
                 worker_env["OMP_NUM_THREADS"] = os.environ["OMP_NUM_THREADS"]
 
+
+            prefix_template = "[${role}${local_rank}]:"
```

Comment:
1. we should really have one place where we default to this prefix template. currently the prefix defaults in 3 places: 
     1. torch.distributed.run (to a sightly different format)
     2. here
     3. TailLogs (we should default it here and you can do this by passing the local_rank -> global rank mapping to TailLogs instead of the substituted prefix for each local rank)
    

5. nit: lets be consistent with the variable name, so this should be `log_line_prefix_template`).

Reply:
I decided to keep it here for two reasons:
1. We can limit the information passed AND most of the contextual data is here, so adding new variables will be simpler
2. I would like to make sure that we always pass header prefix mapping once this lands and we don't run into issues.